lib/rubygems/config_file.rb

```diff
@@ -486,4 +471,21 @@ def ==(other) # :nodoc:
   attr_reader :hash
   protected :hash
 
+  private
+  def set_config_file_name(args)
+    @config_file_name = ENV["GEMRC"] || nil
```

Comment:
I think you can simplify this to

```suggestion
    @config_file_name = ENV["GEMRC"]
```

The only case when I think it would make a difference is when `ENV["GEMRC"]` is explicitly set to `false`, but even in that case, the `load_file` method should shortcircuit in the same way:

https://github.com/rubygems/rubygems/blob/bc6d07d0d9690003c86664b76fb0d8eee5206492/lib/rubygems/config_file.rb#L352

Reply:
Done 👍